projects/core/src/store-finder/occ/store-finder.service.spec.ts

```diff
@@ -6,13 +6,13 @@ import {
 
 import { OccStoreFinderService } from './store-finder.service';
 import { OccConfig } from '@spartacus/core';
-import { SearchConfig } from '../../store-finder/models/search-config';
-import { LongitudeLatitude } from '../../store-finder/models/longitude-latitude';
-import { OccE2eConfigurationService } from '../e2e/e2e-configuration-service';
+import { StoreFinderSearchConfig } from '../../store-finder/model/search-config';
```

Comment:
can be just `from '../model/....'; `

Reply:
`import { StoreFinderSearchConfig } from '../model/search-config';`

Reply:
Import changed.